tests/acceptance/features/webUISharingInternalGroupsToRootEdgeCases/shareWithGroupsEdgeCases.feature

```diff
@@ -4,45 +4,43 @@ Feature: Sharing files and folders with internal groups
   I want to share files and folders with groups
   So that those groups can access the files and folders
 
-  @skip @yetToImplement
-  Scenario Outline: sharing  files and folder with an internal problematic group name
-    Given these users have been created with default attributes:
+  Background:
+    Given these users have been created with default attributes and without skeleton files:
       | username |
       | Alice    |
       | Brian    |
       | Carol    |
+    And user "Carol" has created folder "simple-folder"
+
+  @skip @yetToImplement
```

Comment:
maybe this works by now, if not please open an issue and add this to the expected failures file, let's not use skips  (can also be done in an other PR)

Reply:
Let's do this in other PR because some steps of that test have no step definition and also the same test needs to be adjusted in `webUISharingInternalGroupsEdgeCases`